### DISCUSSION TITLE
Ways to NOT inherit buffer or window variables ?

### DISCUSSION THREAD

**Discussion by asmodeus812** [neovim/neovim#29434] [Q&A]
Hi, i am wondering if neovim, or vim originally, provide any way of not inheriting buffer variables, on new buffer creation. By default it seems that both for windows, and buffers variables are copied from the window/buffer originally being left into the new one

**Comment by adigitoleo** [comment]
What set of commands leads to this behaviour? I don't see buffer variables being copied when doing e.g. `:new`.
